### DISCUSSION TITLE
q-carousel-slide full sized links

### DISCUSSION THREAD

**Discussion by perelin** [quasarframework/quasar#15527] [General - Components / Directives / etc]
Hey, Im currently experimenting with Quasar for a front page with a lot of dynamic content on it.

One of the things I would like to achieve layout wise is a big "Hero" style image slider. I thought `q-carousel` would be ideal for that.

Challenge: I found no way to link a complete `q-carousel-slide` to a another page
```js

          <q-carousel-slide to="/AnotherPage" :name="1" img-src="BigHeroImage">
            <a href="/AnotherPage">
              <div class="absolute-bottom custom-caption">
                <div class="text-h2">HERO Text is important</div>
              </div>
            </a>
          </q-carousel-slide>
```
`to="/AnotherPage"`does nothing
`<a href="/AnotherPage">` works, but only links the inner content of course, not the whole slide

Any ideas? Am I using `q-carousel` wrong? If so: what else to use to achieve a "Hero" effect?

**Comment by MilosPaunovic** [comment]
So, you would like to redirect user to another page on a carousel image change?

**Reply by perelin** [reply]
Hi @MilosPaunovic,
no not on image change, but onClick. Idea is that the whole slide behaves like a button.

**Reply by MilosPaunovic** [reply]
Create a Codepen of what you have so far.
